### PR TITLE
Test azure playwright

### PR DESCRIPTION
**Pull request by linusnorton**
### Change description ###

Use Azure Playwright to execute functional UI tests

### CODE REVIEW

File: src/test/config.ts [hunk 1]
```diff
@@ -148,17 +148,32 @@ export const config = {
...
+    retries: 3,
     waitForNavigation: 'load',
     ignoreHTTPSErrors: true,
     bypassCSP: true,
+    chromium: process.env.PLAYWRIGHT_SERVICE_ACCESS_TOKEN && {
+      timeout: 30000,
+      headers: {
+        'x-mpt-access-key': process.env.PLAYWRIGHT_SERVICE_ACCESS_TOKEN,
+      },
+      exposeNetwork: '<loopback>',
```
Comment (timja) [inline src/test/config.ts:170]: not what you want, it's loopback on your local machine

File: package.json [hunk 2]
```diff
@@ -105,6 +105,7 @@
   },
   "devDependencies": {
     "@codeceptjs/allure-legacy": "1.0.2",
+    "@playwright/test": "1.39.0",
```
Comment (timja) [inline package.json:108]: what's this needed for?

File: package.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
     "test:pa11y": "NODE_TLS_REJECT_UNAUTHORIZED=0 jest -c jest.a11y.config.js --maxWorkers 1",
     "test:cucumber": "yarn test:parallel '^(?!.*@flaky)'",
     "test:e2e": "yarn test:parallel '^(?!.*(@nightly|@flaky))'",
-    "test:parallel": "npx playwright install chromium && NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs run-workers 4 parallel --config ./src/test/functional/codecept.conf.js --grep",
+    "test:parallel": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs run-workers 20 --config ./src/test/functional/codecept.conf.js --grep",
```
[the remark this answers is not in the document]
Comment (linusnorton) [reply package.json:25]: at the time the preview certs were broken, will revert it

File: package.json (hunk 2 above)
Comment (linusnorton) [reply package.json:108]: it's the recommended way now

**Review by timja — APPROVED** [review]

File: Jenkinsfile_CNP [hunk 4]
```diff
@@ -21,7 +21,10 @@ def secrets = [
                 secret('e2e-test-user-password', 'TEST_PASSWORD'),
                 secret('frontend-secret', 'SERVICE_AUTH_SECRET'),
                 secret('idam-secret', 'IDAM_SECRET'),
-                secret('pcq-token-key', 'PCQ_TOKEN')
+                secret('pcq-token-key', 'PCQ_TOKEN'),
+                secret('playwright-service-url', 'PLAYWRIGHT_SERVICE_URL'),
+                secret('playwright-token', 'PLAYWRIGHT_SERVICE_ACCESS_TOKEN'),
+
```
Comment (velimd) [inline Jenkinsfile_CNP:27]: empty space

**Review by velimd — APPROVED** [review]
besides the empty space, it looks good to me

**Pull request merged**